x/emissions/keeper/keeper.go

```diff
@@ -652,6 +651,14 @@ func (k *Keeper) GetParamsMaxLimit(ctx context.Context) (uint64, error) {
 	return params.MaxLimit, nil
 }
 
+func (k *Keeper) GetAlloraExponent() Uint {
```

Comment:
Should eventually move the contents of `x/emissions/keeper/inference_synthesis/common.go` here or replace their usage with this/similar functions.

Reply:
I removed this function